src/redux/store/store.ts

```diff
@@ -35,11 +36,12 @@ const persistConfig = {
   // Blacklist (Don't Save Specific Reducers)
   blacklist: ['nav', 'application', 'communities', 'user'],
   timeout: 0,
-  transforms:[transformCacheVoteMap,transformWalkthroughMap]
+  transforms:[transformCacheVoteMap,transformWalkthroughMap],
+  stateReconciler: autoMergeLevel2,
```

Comment:
What does this do....?

Reply:
actually I was struggling with issue which I have mentioned with you earlier as well. The redux store wasn't reflecting changes when I try to change the reducers structure or add some property/dispatch method. Redux persist has default stateReconciler to autoMergeLevel1 , I had changed it to autoMergeLevel2 so it merges the new changed state and already saved state. U can refer to redux-persist docs here https://github.com/rt2zz/redux-persist.

Reply:
I am entirely sure how it might affect the overall app behaviour internally... @feruzm any thoughts on this.

@aliseyalvi as for reducers to change in persist we usually use spread operator to achieve that, {...state}

Reply:
@noumantahir , @aliseyalvi I checked docs, it says `This acts just like autoMergeLevel1, except it shallow merges two levels`. I don’t think it would break anything but we need to check cache and other recent additions and make sure we won’t have infinite render loops somewhere. Sadaqat maybe go through each page or redux usages quickly and see if you notice something that breaks.

Reply:
Ok, i'll check on my side  as well when checkout the branch, better if u can check on your side by checking this branch, and let me know if anything weired happens.